e2e/_screenshot.js

```diff
@@ -0,0 +1,53 @@
+module.exports = screenshot;
+
+var path = require('path');
+var child_process  = require('child_process');
+var mapnik = require('mapnik');
+
+var SHA = process.env.TRAVIS_COMMIT;
+
+function screenshot(id) {
+  var screenshotPath = path.resolve(__dirname, '..', 'screenshots', id + '.screenshot.png');
+  var screenshotUrl = `https://media.githubusercontent.com/media/angular/visualdiff-dev/${ SHA }/screenshots/${ encodeURIComponent(id) }.screenshot.png`;
+  var newScreenshot;
+
+  browser.takeScreenshot().then(handleNewScreenshot);
+
+  function handleNewScreenshot(png) {
```

Comment:
Nope.  This was shorthand while developing, but I should probably organize this into a class.